src/net/tcp/stream.rs

```diff
@@ -118,39 +134,61 @@ impl TcpStream {
     ///
     /// # Examples
     ///
-    /// ```no_run
-    /// use std::net::SocketAddr;
-    /// use tokio_uring::net::TcpListener;
+    /// ```
```

Comment:
I'll let you figure out why your test and the CI test are different. But this seems to hijack a documentation example for `write_all` with one for `write_zc`. That doesn't seem to fly.